doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+========================================================
+
+This is the XXe Lethe example. It revisits the same taylor-couette flow problem in `Example 2 <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-taylor-couette-flow/2d-taylor-couette-flow.html>`, 
+now using immersed boundaries to represent the inner cylinder. This example demonstrates some of the capabilities of Lethe to simulate the flow around complex geometries 
+ meshing them explicitly with a conformal mesh, but instead by using the Nitsche immersed boundary method available within Lethe.
+
+Features
+----------------------------------
+- Solvers: ``gls_nitsche_navier_stokes_22`` (with Q1-Q1 and Q2-Q1)
+- Steady-state problem
+- Displays the use of an adaptative mesh refinement using Nitsche immersed boundaries
+- Displays the use of the analytical solution to calculate the mesh convergence 
+- Displays the calculation of the torque induced by the fluid on a boundary
+
+
+Location of the examples
+------------------------
+``examples/incompressible_flow/2d_nitsche_taylor-couette/uniform_nitsche_taylor-couette.prm``
+``examples/incompressible_flow/2d_nitsche_taylor-couette/adaptative_nitsche_taylor-couette.prm``
+
+
+Description of the case
+-----------------------
+
+The Taylor-Couette flow is the name of a fluid flow in the gap between two long concentric cylinders with different rotational velocities. 
+We simulate the same case as the regular Taylor-Couette flow where the inner cylinder rotates at a constant angular anti-clockwise velocity :math:`\omega`, 
+while the outer cylinder is fixed. 
+The following figure shows the geometry of this problem and the corresponding boundary conditions. 
+
+.. image:: images/taylor-couette.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+
+Parameter file
+--------------
+
+We first establish the meshes used for the simulation. Using Nitsche immersed boundaries, two meshes are to be defined : the fluid mesh and the geometry mesh (i.e. the inner cylinder).
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+  subsection mesh
+    set type = dealii
+    set grid type = hyper_ball
+    set grid arguments = 0, 0 : 1 : true
+    set initial refinement = 3
+  end
+
+The ``type`` specifies the mesh format used. We use the ``hyper_shell`` mesh generated from the deal.II `GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_ . This GridGenerator generates the mesh of the interstice between two cocentric cylinder. The arguments of this grid type are the position of center of the cylinders (``0, 0``), the inner cylinder radius (`0.25`), the outer cylinder radius (`1`) and the number of subdivision in the azimuthal direction (`4`). All arguments are separated by ``:``. We set ``colorize=true`` and this sets the boundary ID of the inner cylinder to ``0`` and of the outer cylinder to ``1``
```

Comment:
" [...] mesh generated from the deal.II `GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_."
There an extra space between _ and the end of the phrase.
Sorry, my eye caught it quickly :)

"This GridGenerator generates the mesh of the interstice between two concentric cylinders."
Concentric and plural for cylinder.

" [...] and the number of subdivisions in the azimuthal direction [...]"
Subdivisions plural? Not sure :)

Reply:
```suggestion
The ``type`` specifies the mesh format used. We use the ``hyper_shell`` mesh generated from the deal.II `GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_. This GridGenerator generates the mesh of the interstice between two concentric cylinders. The arguments of this grid type are the position of center of the cylinders (``0, 0``), the inner cylinder radius (`0.25`), the outer cylinder radius (`1`) and the number of subdivisions in the azimuthal direction (`4`). All arguments are separated by ``:``. We set ``colorize=true`` and this sets the boundary ID of the inner cylinder to ``0`` and of the outer cylinder to ``1``
```

Reply:
awesome thanks :)